### PR TITLE
Prepare 4.5 Beta 1 Release

### PR DESCRIPTION
**Pull request by jonathansadowski**
### Changes proposed in this Pull Request:

* Update version in `woocommerce.php`.
* Add new entries for 4.5.0 to `changelog.txt`.
* Remove 4.4.1 entries from `readme.txt` and add 4.5.0 entries.

### CODE REVIEW

File: changelog.txt [hunk 1]
```diff
@@ -1,5 +1,40 @@
 == Changelog ==
 
+= 4.5.0 - 2020-09-08 =
+
+**WooCommerce**
+* Localization - Added postcode validation for Bosnia and Herzegovina. #27048
+* Localization - Added the postcode validation for Liechtenstein. #27059
+* Localization - Add i18n locale information for Liechtenstein, Switzerland and Austria. #27193
+* Fix - Added additional stock-based cart filters including woocommerce_cart_product_cannot_add_another_message, woocommerce_cart_product_out_of_stock_message and woocommerce_cart_product_not_enough_stock_message. #26439
```
Comment (claudiosanches) [inline changelog.txt:9]:
```suggestion
* Dev - Added additional stock-based cart filters including `woocommerce_cart_product_cannot_add_another_message`, `woocommerce_cart_product_out_of_stock_message` and `woocommerce_cart_product_not_enough_stock_message`. #26439
```

**Comment by rrennick** [comment]
@jonathansadowski Do we need to update `package.json` with `4.5.0-beta-1` ?

**Review by claudiosanches — APPROVED** [review]
Looks great!

**Comment by claudiosanches** [comment]
@rrennick on the `package.json` we update with beta or RC. Unless it's required to publish any package for e2e, so we can also do it if you like.

**Pull request merged**